spid_cie_oidc/onboarding/tests/token_request_settings.py

```diff
@@ -47,3 +47,59 @@
 
 TOKEN_REFRESH_REQUEST_NO_CORRECT_GRANT_TYPE = deepcopy(TOKEN_REFRESH_REQUEST)
 TOKEN_REFRESH_REQUEST_NO_CORRECT_GRANT_TYPE["grant_type"] = "authorization_code"
+
+JWT_CLIENT_ASSERTION = {
+    "iss": "https://idserver.servizicie.interno.gov.it/op/",
+    "sub": "https://idserver.servizicie.interno.gov.it/op/",
```

Comment:
Mmhh no

Iss and sub must match to client_id of the rp that makes the request

Reply:
ok, I have modified